CHANGES.md

```diff
@@ -3,9 +3,12 @@ Change Log
 
 ### MobX Development
 
-#### next release (8.0.0-alpha.59)
+#### next release (8.0.0-alpha.60)
```

Comment:
We don't need this, it's a bit confusing - the next release is .59

Reply:
fixed

Reply:
Ok what I meant is, keep the heading `#### next release (8.0.0-alpha.59)`, and remove the heading `#### next release (8.0.0-alpha.60)` - we haven't released 8.0.0-alpha.59 yet 🙂

Reply:
Like so: https://github.com/TerriaJS/terriajs/pull/4918/commits/faba6347d473bfb5e6f0c37d68700d93e1673586